docs/content/guides/dagster/transitioning-data-pipelines-from-development-to-production.mdx

```diff
@@ -390,16 +489,6 @@ class StubHNClient:
         return ["id", "type", "title", "by"]
 ```
 
-<Note>
-  Since the stub Hacker News resource and the real Hacker News resource need to
-  implement the same methods, this would be a great time to write an interface.
-  We’ll skip the implementation in this guide, but you can find it in the{" "}
-  <a href="https://github.com/dagster-io/dagster/tree/master/examples/development_to_production">
-    full code example
-  </a>
-  .
-</Note>
-
 Now we can use the stub Hacker News resource to test that the `items` asset transforms the data in the way we expect:
 
 ```python file=/guides/dagster/development_to_production/test_assets.py startafter=start endbefore=end
```

Comment:
[1] the code snippet here where we have the `test_items` unit test won't work anymore because the asset is writing to a snowflake table instead of returning the data. 

```python
def test_items():
    test_snowflake = SnowflakeResource(
            account="abc1234.us-east-1",
            user="test@company.com",
            password=os.getenv("TEST_SNOWFLAKE_PASSWORD"),
            database="TEST",
            schema="HACKER_NEWS",
        ),
    try:
        items(
            config=ItemsConfig(base_item_id=StubHNClient().fetch_max_item_id()),
            hn_client=StubHNClient(),
            snowflake_resource=test_snowflake 
        )
        with test_snowflake.get_connection() as conn:
            hn_dataset = conn.cursor.execute("select * from ITEMS").fetch_pandas_all()
            assert isinstance(hn_dataset, pd.DataFrame)
            expected_data = pd.DataFrame(StubHNClient().data.values()).rename(
                columns={"by": "user_id"}
            )
            assert (hn_dataset == expected_data).all().all()

    finally:
          with test_snowflake.get_connection() as conn:
               conn.cursor.execute("drop table if exists ITEMS")
```

I think we should do this instead. Sorry the indenting is all off, moving this snippet into github messed things up

Reply:
just to follow up on this - i wrote this without testing it, so i will take a minute today to make sure it works